src/ckeditor.js

```diff
@@ -46,11 +45,15 @@ export default {
 		throttle: {
 			type: Number,
 			default: 80
+		},
+		namespaceLoaded: {
+			type: Function,
+			default: () => {}
 		}
 	},
 
 	mounted() {
-		getEditorNamespace( this.editorUrl ).then( () => {
+		getEditorNamespace( this.editorUrl, this.namespaceLoaded ).then( namespace => {
```

Comment:
Is passing `namespace` necessary here? It isn't used and tests are still passing without it.

Reply:
No, it's not needed.